languages/core_de.php

```diff
@@ -1 +1 @@
 Sever gilt.";$lang['wits3dch']="Default Channel";$lang['wits3dchdesc']="Die channel-ID, mit der sich der Bot verbindet.<br><br>In diesem Channel wechselt der Bot nach dem Verbinden mit dem TeamSpeak Server.";$lang['wits3host']="TS3 Hostadresse";$lang['wits3hostdesc']="TeamSpeak 3 Server Adresse<br>(IP oder DNS)";$lang['wits3sm']="Slowmode";$lang['wits3smdesc']="Mit dem Slowmode werden die Query-Anfragen an den TeamSpeak Server reduziert. Dies schützt vor einem Ban aufgrund von flooding.<br>TeamSpeak Query Befehle werden mit dieser Funktion verzögert abgeschickt.<br><br>!!! AUCH REDUZIERT ER DIE CPU USAGE !!!<br><br>Die Aktivierung ist nicht empfohlen, wenn nicht benötigt. Die Verzögerung (delay) erhöht die Laufzeit eines Durchgangs des Bots, dadurch wird er unpräziser.";$lang['wits3qnm']="Botname";$lang['wits3qnm2']="2. Botname";$lang['wits3qnm2desc']="Ein zweiter Botname, falls der erste bereits genutzt wird.";$lang['wits3qnmdesc']="Der Name, mit dem die Query-Verbindung augebaut werden soll.<br>Der Name kann frei gewählt werden.";$lang['wits3querpw']="TS3 Query-Passwort";$lang['wits3querpwdesc']="TeamSpeak 3 Query Passwort<br>Passwort des Query Benutzers";$lang['wits3querusr']="TS3 Query-Benutzer";$lang['wits3querusrdesc']="TeamSpeak 3 Query Benutzername<br>Standard ist serveradmin<br>Natürlich kann auch ein gesonderter Query-Benutzer erstellt und genutzt werden.<br>Die benötigten Rechte sind hier aufgelistet:<br>http://ts-n.net/ranksystem.php";$lang['wits3query']="TS3 Query-Port";$lang['wits3querydesc']="TeamSpeak 3 Query Port<br>Standard ist 10011 (TCP)<br>andere Werte sollten in der 'ts3server.ini' zu finden sein.";$lang['wits3voice']="TS3 Voice-Port";$lang['wits3voicedesc']="TeamSpeak 3 Voice Port<br>Standard ist 9987 (UDP)<br>Dieser Port wird zum Verbinden mit dem TS3 Client genutzt.";$lang['witxco']="Textfarbe:";$lang['witxcodesc']="Lege eine Textfarbe fest.<br>(gültiger HTML Code; muss mit # beginnen)";$lang['wiupcheck']="Update-Prüfung";$lang['wiupcheckdesc']="Ist die Update-Prüfung aktiv, werden nachfolgend genannte User mit einer privaten Textnachricht informiert, sobald ein Update verfügbar ist.";$lang['wiuptime']="Prüfungsintervall";$lang['wiuptimedesc']="Gib an alle wie viel Sekunden das Ranksystem prüfen soll, ob ein Update verfügbar ist.<br>Beachte, bei jeder Prüfung werden die hinterlegten User über ein verfügbares Update benachrichtigt. Ist kein hinterlegter User online, wird mit dem nächsten Intervall versucht zu benachrichtigen.";$lang['wiupuid']="Empfänger";$lang['wiupuiddesc']="Eine Komma getrennte Liste von eindeutigen Client-IDs, welche via privater Textnachricht auf dem TeamSpeak über ein Update informiert werden sollen.";$lang['wiversion']="(verwendete Version %s)";$lang['wivlang']="Sprache";$lang['wivlangdesc']="Sprache des Ranksystems<br><br>de - Deutsch<br>en - english<br>ru - русский";$lang['wiwnco']="Warntextfarbe:";$lang['wiwncodesc']="Lege eine Warntextfarbe fest.<br>(gültiger HTML Code; muss mit # beginnen)";?>
```

Comment:
"Eine *mit* Komma getrennte Liste von Servergruppen-IDs, welche nicht am Ranksystem teilnehmen sollen.<br><br>User in mindestens einer dieser Gruppen sind von Rangsteigerungen ausgenommen.";$lang['wiexuid']="Client-Ausnahmen";$lang['wiexuiddesc']="Eine *mit* Komma getrennte Liste von eindeutigen Client-IDs, welche nicht am Ranksystem teilnehmen sollen.<br><br>Aufgelistete User sind von Rangsteigerungen